ArduCopter/autoyaw.cpp

```diff
@@ -123,12 +123,10 @@ void Mode::AutoYaw::set_fixed_yaw(float angle_deg, float turn_rate_dps, int8_t d
         // default to regular auto slew rate
         _fixed_yaw_slewrate = AUTO_YAW_SLEW_RATE;
     } else {
-        const int32_t turn_rate = (wrap_180_cd(_fixed_yaw - curr_yaw_target) / 100) / turn_rate_dps;
-        _fixed_yaw_slewrate = constrain_int32(turn_rate, 1, 360);    // deg / sec
+        _fixed_yaw_slewrate = wrap_360_cd(turn_rate_dps) * 100;
     }
 
-    // set yaw mode
-    set_mode(AUTO_YAW_FIXED);
+    set_rate(_fixed_yaw_slewrate);
```

Comment:
The original will hold the head in place at the final angle.
I don't think this change fixes the head.